send/send.h

```diff
@@ -35,7 +35,7 @@ struct EmailArray;
 struct Envelope;
 struct Mailbox;
 
-typedef uint16_t SendFlags;             ///< Flags for mutt_send_message(), e.g. #SEND_REPLY
+typedef uint32_t SendFlags;             ///< Flags for mutt_send_message(), e.g. #SEND_REPLY
```

Comment:
and this one, your code couldn't be run.
Leading me to conclude that you hadn't tested it.

Reply:
If I remember well, I did test it, but maybe I didn't check the results well...  A bit embarrasing!